adding_species_manuscript.tex

```diff
@@ -540,12 +541,13 @@ \subsection*{Basic setup for chromosome-level simulations}
 Therefore, its incorporation into simulations is crucial for many purposes.
 To achieve this, the simulator needs to know which regions along the genome are subject to selection,
 and the nature and strength of this selection.
-The current version of \stdpopsim enables simulation with selection
+The development version of \stdpopsim enables simulation with selection
 (using the \texttt{SLiM} engine)
 by specifying genome annotations and distributions of fitness effects,
 as specified below.
-We note that the ability to simulate chromosomes with realistic models of
-selection is still under development and will be finalized in the next release of \stdpopsim.
+The ability to simulate chromosomes with realistic models of
+selection is still under development and will be finalized in the next release of \stdpopsim,
+and will be described in more detail in a future paper.
```

Comment:
```suggestion
The ability to simulate chromosomes with realistic models of
selection is still under development. It will be finalized in the next release of \stdpopsim,
and described in more detail in a future paper.
```